src/api.py

```diff
@@ -7,50 +7,53 @@
 from src.tables import APIKey
 from src.db import session_local, engine
 
+# Initialize FastAPI app
 app = FastAPI()
+
+# Event handler for startup to create database tables
 @app.on_event("startup")
```

Comment:
This code block (closing the database session) can be abstracted into context managers or decorators to minimize code repetition and improve DRY compliance.